Classes/PhoneMainView.m

```diff
@@ -677,15 +677,15 @@ - (void)playMessageSoundForCallID:(NSString *)callID {
 }
 
 - (void)displayIncomingCall:(LinphoneCall *)call {
-//	LinphoneCallLog *callLog = linphone_call_get_call_log(call);
-//	NSString *callId = [NSString stringWithUTF8String:linphone_call_log_get_call_id(callLog)];
+	LinphoneCallLog *callLog = linphone_call_get_call_log(call);
+	NSString *callId = [NSString stringWithUTF8String:linphone_call_log_get_call_id(callLog)];
```

Comment:
Note: We had commented this out. I wonder if this will introduce some call log crash.